apps/AnnotateVariants/Program.cs

```diff
@@ -207,19 +336,21 @@ static void Main(string[] args)
                 }
             }
 
-            genome.load(configuration.indexDirectory);
+			int nCasesToProcess = casesToProcess.Count();
+			//  genome.load(configuration.indexDirectory);
```

Comment:
I am not using the genome here, and pulling from the MAF refs to compare whether a read matches the ref or alt. Is that wrong?